brave/browser-laptop
# Fix Pocket setting l10

Fixes #7450

Test Plan: (requires up-to-date translations)
1. Change to a language (Preferences > General > Language) with a up-to-date translation for `enableExtension` (see `en-US/preferences.properties` for an example).
2. Open Preferences > Advanced.
3. Find the setting `Enable Pocket` under Extensions.
4. Make sure the setting matches "Pocket" exactly and is not translated.
